packages/core/src/mutation/__tests__/create_json_mutation.test.ts

```diff
@@ -143,7 +146,10 @@ describe('createJsonMutation', () => {
     expect(listeners.onSuccess).toHaveBeenCalled();
     expect(listeners.onSuccess).toHaveBeenCalledWith(
       expect.objectContaining({
-        result: null,
+        result: {
```

Comment:
🚨 It is breaking change, and I do not see any reasons to do it 🤔 

We can just use `{ result, meta: { headers }, params }`, it won't break user's code and will work well for your use cases.